chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -159,6 +159,7 @@
 if node[:platform_family] == "suse"
```

Comment:
The commit message says "load on non SUSE systems", but this code says otherwise. Which one is correct?